configs/run2_Z_2017_miniIso.json

```diff
@@ -0,0 +1,58 @@
+{
+    "efficiencies": [
+        ["TightIDandMiniIso", "TightID"],
+        ["LooseIDandMiniIso", "LooseID"]
+    ],
+    "selection": "tag_pt>29 and tag_abseta<2.4 and tag_IsoMu27==1 and pair_probeMultiplicity==1 and tag_combRelIsoPF04dBeta<0.2",
+    "definitions": {
+        "LooseID": "CutBasedIdLoose == 1",
+        "TightID": "CutBasedIdTight == 1",
+        "MiniIsolationLoose" : "MiniIsolation < 0.4",
+        "MiniIsolationTight" : "MiniIsolation < 0.1",
+        "LooseIDandMiniIso" : "CutBasedIdLoose == 1 and MiniIsolation < 0.4",
```

Comment:
The "CutBasedIdLoose == 1" is redundant here since it is already required in the denominator of the efficiencies.